libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -375,8 +375,8 @@ - (void) syncUp:(SFSyncState*)sync updateSync:(SyncUpdateBlock)updateSync failSy
     // Call smartstore
     NSArray* dirtyRecordIds = [target getIdsOfRecordsToSyncUp:self soupName:soupName];
     NSUInteger totalSize = [dirtyRecordIds count];
-    updateSync(nil, totalSize == 0 ? 100 : 0, totalSize, kSyncManagerUnchanged);
```

Comment:
We introduced that line a couple of days ago - which meant one extra update than previously when there are changes.